packages/lab/src/form-field-next/FormFieldNext.tsx

```diff
@@ -17,15 +15,7 @@ export interface FormFieldProps
    */
   disabled?: boolean;
   /**
-   * The helper text content
-   */
-  helperText?: string;
```

Comment:
would I need to deprecate these given how recent they were added? @joshwooding

Reply:
Removing these is fine